docs/source/documentation.rst

```diff
@@ -231,17 +214,17 @@ The script to launch is called **axondeepseg**. It takes several arguments:
                     Type of acquisition to segment.
                     SEM: scanning electron microscopy samples. 
                     TEM: transmission electron microscopy samples.
-                    OM: bright field optical microscopy samples.
+                    BF: bright field optical microscopy samples.
 
 -i IMGPATH
                     Path to the image to segment or path to the folder where the image(s) to segment is/are located.
 
 **Optional arguments:**
 
 -m MODEL            Folder where the model is located. 
-                    The default SEM model path is **default_SEM_model**. 
-                    The default TEM model path is **default_TEM_model**.
-                    The default OM model path is **model_seg_pns_bf**.
+                    The default SEM model path is **model_seg_rat_axon-myelin_sem**. 
+                    The default TEM model path is **model_seg_mouse_axon-myelin_tem**.
+                    The default BF model path is **model_seg_rat_axon-myelin_bf**.
 
 -s SIZEPIXEL        Pixel size of the image(s) to segment, in micrometers. 
                     If no pixel size is specified, a **pixel_size_in_micrometer.txt** file needs to be added to the image folder path ( that file should contain a single float number corresponding to the resolution of the image, i.e. the pixel size). The pixel size in that file will be used for the segmentation.
```

Comment:
Verbose description at line 232 below to adjust to new behavior.

Reply:
Resolved in c9eef2098